package.json

```diff
@@ -42,6 +42,8 @@
   },
   "devDependencies": {
     "@actions/core": "^1.10.1",
+    "@types/bun": "^1.1.14",
+    "@types/mocha": "^10.0.6",
```

Comment:
Why we need mocha?

Reply:
no need mocha, no test was implemented in this branch so there is a lot of unclean stuff @nazarhussain 
we're trying to run beacon-node/validator against public network on this branch and resolve issues along the way, kind of top-down approach
once you support testing with bun in `unstable` I'll merge here to resolve these issues